articles/overview/what-is-quantum-computing.md

```diff
@@ -0,0 +1,46 @@
+---
+title: What is quantum computing?
+description: 
+author: natke
+ms.author:  
+ms.date: 10/14/2019
+ms.topic: article
+uid: microsoft.quantum.overview.whatis
+---
+
+# What is quantum computing?
+
+Quantum computing is the application of quantum physics to the field of computer science. Quantum computing might sound daunting, but with the right resources you can start building quantum applications today.
+
+## The qubit
+
+The basic unit of quantum computing is the `qubit`. Unlike a `bit` in classical computing, which can be `0` or `1`, a qubit can be in an infinite number of states - a **superposition** of both 0 and 1.
+
+The act of **measurement** changes a qubit. It goes from being in the infinite states of superposition to one of the binary states of 0 or 1. Again, measurement of a qubit is unlike a classical bit, which when 0 is measured as 0, and when 1 is measured as 1.
+
+Qubits can also be **entangled**: when we make a measurement of one entangled qubit, it affects the state of the other, even if the qubits are physically far apart.
+
```

Comment:
At this point, superposition, measurement, and entanglement probably feel a bit mysterious, such that it might be helpful emphasizing that we can understand these quantum effects through working with the Quantum Development Kit or through looking at the math, whichever we prefer.

Reply:
Yes, I agree! I would love to come up with a terse example but have so far failed.

Reply:
Let me think a bit on that...

Reply:
I agree, interference and entanglement are concepts difficult to grasp. I don't know if we should adress them in the introduction, they can be confusing and mysterious. But on the other hand they are very important to understand the reason of quantum computing.